vercel.json

```diff
@@ -0,0 +1,755 @@
+      "permanent": true
+    },
+    {
+      "source": "/faq/",
+      "destination": "/docs/introduction/faq",
+      "permanent": true
+    },
+    {
+      "source": "/docs/tina-cloud/faq/",
+      "destination": "/docs/introduction/faq",
+      "permanent": true
+    },
+    {
+      "source": "/cloud/",
+      "destination": "/",
+      "permanent": true
+    },
+    {
+      "source": "/blog/tina-v-0.69.0/",
+      "destination": "/blog/tina-v-0.68.14",
+      "permanent": true
+    },
+    {
+      "source": "/docs/extending-tina/customizing-forms/",
+      "destination": "/docs/extending-tina/overview/",
+      "permanent": true
+    },
+    {
+      "source": "/docs/tina-cloud/alpha-faq/",
+      "destination": "/docs/tina-cloud/faq/",
+      "permanent": true
+    },
+    {
+      "source": "/blog/first/",
+      "destination": "/blog/announcing-tinacms",
+      "permanent": true
+    },
+    {
+      "source": "/blog/introducing-tina-grande-%F0%9F%8E%89/",
+      "destination": "/blog/introducing-tina-grande",
+      "permanent": true
+    },
+    {
+      "source": "/jamstack-conf/",
+      "destination": "/teams",
+      "permanent": true
+    },
+    {
+      "source": "/smashing-conf/",
+      "destination": "/teams",
+      "permanent": true
+    },
+    {
+      "source": "/docs/concepts/sidebar/",
+      "destination": "/docs/cms",
+      "permanent": true
+    },
+    {
+      "source": "/docs/concepts/forms/",
+      "destination": "/docs/forms",
+      "permanent": true
+    },
+    {
+      "source": "/docs/plugins/forms/",
+      "destination": "/docs/forms",
+      "permanent": true
+    },
+    {
+      "source": "/docs/concepts/fields/",
+      "destination": "/docs/fields",
+      "permanent": true
+    },
+    {
+      "source": "/docs/cms/apis/",
+      "destination": "/docs/apis",
+      "permanent": true
+    },
+    {
+      "source": "/docs/ui/",
+      "destination": "/docs/sidebar-toolbar",
+      "permanent": true
+    },
+    {
+      "source": "/docs/cms/ui/",
+      "destination": "/docs/sidebar-toolbar",
+      "permanent": true
+    },
+    {
+      "source": "/docs/cms/alerts/",
+      "destination": "/docs/ui/alerts",
+      "permanent": true
+    },
+    {
+      "source": "/docs/cms/events/",
+      "destination": "/docs/events",
+      "permanent": true
+    },
+    {
+      "source": "/docs/cms/plugins/",
+      "destination": "/docs/plugins",
+      "permanent": true
+    },
+    {
+      "source": "/docs/plugins/fields/",
+      "destination": "/docs/fields",
+      "permanent": true
+    },
+    {
+      "source": "/docs/plugins/fields/",
+      "destination": "/docs/fields",
+      "permanent": true
+    },
+    {
+      "source": "/docs/concepts/backends/",
+      "destination": "/docs/cms#apis",
+      "permanent": true
+    },
+    {
+      "source": "/docs/concepts/plugins/",
+      "destination": "/docs/cms#plugins",
+      "permanent": true
```

Comment:
the fragment identifiers aren't doing anything here

Reply:
thats because the identifiers (based on titles) dont exist anymore, updatign to remove the identifiers